css/module.css

```diff
@@ -0,0 +1,66 @@
+input[type="text"],
```

Comment:
In this document you're targeting many element selectors, usually these will be used for base styles. Modules usually consist of 2 or more base elements. Modules on the other hand are going to be constructed using classes that follow a naming convention like BEM. Please refactor your modules to reflect this. Let me know if you have any questions.